### ISSUE TITLE
`readalongs -h` confusing output: flask, angular (?) and readalongs CLI conflated

### ISSUE THREAD

**Issue by joanise** [ReadAlongs/Studio#31]
When running `readalongs -h`, the output combines and conflates CLI elements added by readalongs itself with those added by flask and, I believe, angular.

The `--version` option appears twice - the first comes from flask, the second from readalongs:
```
  --version   Show the flask version
  --version   Show the version and exit.
```

As for the five commands shown,
 - `align` and `epub` are defined in `readalongs/cli.py`.
 - `routes` appears to come from angular - is it relevant? Is the output `readalongs routes` produces meaningful?
 - `run`: I'm not sure where this comes from, since I cannot find the strong 'Runs a development server.' anywhere in the code base, but we use it, so it's clearly relevant. But where does it come from?
 - `shell`: again, not sure where this is defined. I've never used it, what is it for exactly?

**Comment by roedoejet** [comment]
We don't use Angular in this repo, but these things are coming from [click](https://click.palletsprojects.com/en/7.x/).

Specifically, from using the default `FlaskGroup`:

```python
@click.version_option(version=__version__, prog_name="readalongs")
@click.group(cls=FlaskGroup, create_app=create_app, context_settings=CONTEXT_SETTINGS)
def cli():
    """Management script for Read Along Studio."""
```

This is a way of wrapping up a [Flask application into a command line application](https://flask.palletsprojects.com/en/1.1.x/cli/). The commands it provides by default are the `routes`, `run`, and `shell` commands. 

- `readalongs routes` shows all the available routes in the flask app. I think this is helpful.
- `readalongs run` runs the flask app. This is also helpful.
- `readalongs shell` opens a shell within the flask application context. This can also be somewhat helpful, but I don't use it that often.

Basically, I think everything can stay the same, except maybe we can overwrite the `FlaskGroup` version method.

**Comment by roedoejet** [comment]
Ah, it looks like I did overwrite the version option: `@click.version_option(version=__version__, prog_name="readalongs")` but the default is still showing up in the help.

**Comment by joanise** [comment]
OK, so all the commands do make sense, thanks for the explanation.

The only thing I would fix, then, is that `--version` should appear just once in the help, and maybe it should print both the readalongs version and the flask version, if that latter part is relevant, or only the readalongs version otherwise.

**Comment by roedoejet** [comment]
Fixed as of https://github.com/ReadAlongs/Studio/pull/90
